.gitlab-ci.yml

```diff
@@ -0,0 +1,105 @@
+##############################################################################
+# Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and Axom
+# project contributors. See the LICENSE file for details.
+##############################################################################
+
+variables:
+  GIT_SUBMODULE_STRATEGY: recursive
+  PROJECT_ALLOC_NAME: ${CI_PROJECT_NAME}_ci_${CI_PIPELINE_ID}
+  BUILD_ROOT: ${CI_PROJECT_DIR}
+  SLURM_ACCOUNT: -p pdebug
+  LSF_ACCOUNT: -q pdebug -G eng
```

Comment:
-G eng is wrong here

Reply:
Though I'm now getting this with the quartz builds:

```
srun: error: Unable to allocate resources: Invalid account or account/partition combination specified
```

Reply:
I thought I got all of those. have to look again.

Reply:
the quartz one should have used the SLURM_ACCOUNT which worked a week ago =/